app/grants/models.py

```diff
@@ -72,6 +72,19 @@ def keyword(self, keyword):
 class Grant(SuperModel):
     """Define the structure of a Grant."""
 
+    GRANT_CATEGORIES = [
```

Comment:
pls DRY this up

Reply:
This is the only occurrence in the djanago backend. How should I DRY this up? Thanks :)

Reply:
@vince0656 theres a list of categories here, as well as in the frontend code - if one changes, then the other needs to change as well. This is generally an anti-pattern, both should inherit this data such that if it changes in the backend, the frontend should automatically support this. I recommend a helper API endpoint that just spits out the current list of the supported categories such that the frontend can ping this endpoint and easily have an up-to-date list.

Reply:
@danlipert got it! I'll call the endpoint from `shared.js`. Thanks 👍